main.js

```diff
@@ -0,0 +1,104 @@
+const time = document.getElementById('time');
+const greet = document.getElementById('greet');
+const answer = document.getElementById('answer');
+const name = document.getElementById('name');
+
+function showTime() {
+    let currenTime = new Date(),
+        hour = currenTime.getHours(),
+        minute = currenTime.getMinutes(),
+        secund = currenTime.getSeconds();
+        pmAM = '';
+    pmAM = hour < 12 ? 'PM' : 'AM';
+    hour = hour % 12 || 12;
+
+    if(secund < 10) {
+        secund = `0${secund}`;
+    }
+    if(minute < 10) {
+        minute = `0${minute}`;
+    }
+    if(hour < 10) {
+        hour = `0${hour}`;
+    }
+    time.innerHTML = `${hour} : ${minute} : ${secund} ${pmAM}`;
+    setTimeout(showTime, 1000);
+}
+
+function changeBackGroundGreet() {
+    let currenTime = new Date(),
+        hour = currenTime.getHours();   
+    if(hour < 12) {
+        document.body.style.backgroundImage = "url('https://i.ibb.co/7vDLJFb/morning.jpg')";
+        greet.innerHTML = 'Good Morning, ';
+    }else if(hour < 18) {
+        document.body.style.backgroundImage = "url('https://i.ibb.co/3mThcXc/afternoon.jpg')";
+        greet.innerHTML = 'Good Afternoon, ';
+
+    }else {
+        document.body.style.backgroundImage = "url('https://i.ibb.co/924T2Wv/night.jpg')";
+        document.body.style.color = "white";
+        greet.innerHTML = 'Good Evening, ';
+    }
+
+}
+
+function getName() {
+    if(!localStorage.getItem('name')) {
+        name.innerHTML = '[Enter Name]';
+    }else {
+        name.innerHTML = localStorage.getItem('name');
+    }
+}
+
+function getAnswer() {
+    if(!localStorage.getItem('answer')) {
+        answer.innerHTML = '[Enter Answer]';
+    }else {
+        answer.innerHTML = localStorage.getItem('answer');
+    }
+}
+
+function setName(event) {
+    if (event.type === 'keypress') {
+      if (event.keyCode == 13) {
```

Comment:
better use strict equality and also I'd combine these 2 conditions like:

`if (event.type === 'keypress' && event.keyCode == 13)`